app/controllers/application_controller.rb

```diff
@@ -134,4 +134,12 @@ def upcoming_workshops
   end
 
   helper_method :upcoming_workshops
+
+  def redirect_back(fallback_location:, **args)
```

Comment:
Once it will be [Rails 5][1], this should be deleted because is already [there][2].

[1]: https://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html
[2]: https://github.com/rails/rails/pull/22506

Reply:
Sorry, my fault I missed that.